src/Uno.Toolkit.RuntimeTests/Tests/AutoLayoutTest.cs

```diff
@@ -77,6 +77,20 @@ public async Task When_SpaceBetween_with_spacing(Orientation orientation, double
 	}
 
 	[TestMethod]
+	//Element with border in wasm don't give the same result for TransformToVisual
+	//https://github.com/unoplatform/uno/issues/11410
+#if NETSTANDARD2_0
```

Comment:
we dont have that in a library project(which this runtimetests is), because wasm and skia-ish all gets compiled to netstandard2.0 assembly

use the following to make a "runtime conditional":
- drop the #ifs and keep the original `[DataRow]`s
- adjust the code:
	```
	// Element with border in wasm don't give the same result for TransformToVisual
	// https://github.com/unoplatform/uno/issues/11410
	var offset = RuntimeInformation.IsOSPlatform(OSPlatform.Create("BROWSER")) ? 2 : 0;
	
	//...
	
	Assert(value, rec1expected + offset);
	Assert(value, rec2expected + offset);
	```

Reply:
@Xiaoy312 done.